compute/instances/creating-instance-with-custom-machine-type/custom_machine_type_helper_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestCustomMachineTypeSnippets(t *testing.T) {
+	zone := "europe-central2-b"
+
+	// N1
+	cmt1, err := createCustomMachineType(zone, N1, 8192, 8, CPUSeries_N1_Limit)
```

Comment:
These tests look like a great candidate for [table-driven testing](https://github.com/golang/go/wiki/TestComments#table-driven-tests-vs-multiple-test-functions)